modules/logging-rn-5.7.2.adoc

```diff
@@ -0,0 +1,115 @@
+
+* Before this change when configuring log forwarding to write to a Kafka output topic which matched a key in the payload, logs dropped due to an error.  With this update, Fluentd's buffer name has been prefixed with an underscore, resolving the issue.(link:https://issues.redhat.com/browse/LOG-4027[LOG-4027])
+
+* Before this update, the LokiStack gateway returned label values for namespaces without applying the access rights of a user. With this update, the LokiStack-gateway applies permissions to label value requests, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4049[LOG-4049])
+
+* Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API will no longer require a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
++
+[source,yaml]
+----
+tls.verify_certificate = false
+tls.verify_hostname = false
+----
++
+(link:https://issues.redhat.com/browse/LOG-3445[LOG-3445])
+
+* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to timeout. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4052[LOG-4052])
+
+* Before this update, a prior fix to remove defaulting of the `collection.type` resulted in the Operator no longer honoring the deprecated specs for resource, node selections, and tolerations.  This update modifies the Operator behavior to always prefer prefer the `collection.logs` spec over those of `collection`.  Note this varies from previous behavior that allowed using both the preferred fields and deprecated fields but would ignore the deprecated fields when `collection.type` was populated. (link:https://issues.redhat.com/browse/LOG-4185[LOG-4185])
+
+* Before this update, the Vector log collector did not generate TLS configuration for forwarding logs to multiple Kafka brokers if the broker URLs were not specified in the output. With this update, TLS configuration is generated appropriately for multiple brokers. (link:https://issues.redhat.com/browse/LOG-4163[LOG-4163])
+
+* Before this update, the option to enable passphrase for log forwarding to Kafka was unavailable. This limitation presented a security risk as it could potentially expose sensitive information. With this update, users now have a seamless option to enable passphrase for log forwarding to Kafka. (link:https://issues.redhat.com/browse/LOG-3314[LOG-3314])
+
+* Before this update, Vector log collector did not honor the `tlsSecurityProfile `settings for outgoing TLS connections. After this update, Vector handles TLS connection settings appropriately. (link:https://issues.redhat.com/browse/LOG-4011[LOG-4011])
+
```

Comment:
I think you can delete "now" from the clause "With this update, metrics now contain..."

Reply:
Not a single one of your comments seems to be associated with the appropriate lines you're referring to, FYI.